### PR TITLE
Pt br

### PR DESCRIPTION
**Pull request by FKrauss**
Adding Brazilian portuguese to the website, following many comments and a some restructuring that was required from the [previous PR about brazilian portuguese](https://github.com/bisq-network/bisq-website/pull/281)

### CODE REVIEW

File: _data/pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  bisqDaoBrief: A DAO Bisq em pequenos vídeos
+  pBisqDaoBrief: "<p class=\"bigger\">Série de vídeos <a href=\"https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e\" target=\"_blank\">curtos e rápidos</a> sobre a DAO Bisq.</p>"
+  technicalOverview: Visão Geral Técnica - Doc
+  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">Especificação</a> do funcionamento da DAO Bisq e do token BSQ."
+  notConvinced: "Não está convencido de que um token é necessário?"
+  pNotConvinced: "<p class=\"bigger\">Nós temos um <a href=\"https://bisq.network/blog/bisq-dao-for-bitcoin-maximalists/\" target=\"_blank\">post</a> só para você.</p>"
+  seeMore: >
+    <p>Veja mais recursos em nosso<a href='https://docs.bisq.network/dao.html'> Tudo Sobre a DAO Bisq</a> doc.</p><p>Contribuidores anteriores, verifiquem<a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>o link</a>para obter instruções sobre como ser adicionado à distribuição de BSQ da gênese.</p>
```
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
    <p>Veja mais recursos em nosso<a href='https://docs.bisq.network/dao.html'> Tudo Sobre a DAO Bisq</a> doc.</p><p>Contribuidores anteriores, verifiquem <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>o link</a> para obter instruções sobre como ser adicionado à distribuição de BSQ da gênese.</p>
```

Missing spaces around link.

File: _data/pt-BR.yml [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+      <li>Transferência para o mesmo banco</li>
+      <li>Transferência para banco específico</li>
+      <li>Uphold</li>
+      <li>US Postal Money Order</li>
+      <li>WeChat Pay</li>
+      <li>Western Union</li>
+      <li>Zelle (ClearXchange)</li>
+    </ul> <p>Por favor veja nosso <a href="https://docs.bisq.network/payment-methods" target="_blank"> document sobre meios de pagamento</a> para mais detalhes. Cada meio de pagamento tem seu prório período de negociação e limites.</p>
+  hTradingDetails: Detalhes da negociação
+  hTradeLimits: "Quais são os limites de negociação?"
```
Comment (m52go) [inline _data/pt-BR.yml:182]: This FAQ isn't updated for v1.2.

See:
https://bisq.network/faq#trade-limits

File: _data/pt-BR.yml [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
...so é opcional, mas existem <a href="https://docs.bisq.network/getting-started-dao-traders.html" target="_blank">benefícios significativos</a>.</p> <div id="faq-tables"> <p class="note">Taxas de negociação de BTC, por 1 BTC negociado:</p> <table> <thead> <tr> <th></th> <th> Taxas em BTC</th> <th>Taxa como % da transação no valor de 1 BTC</th> </tr> </thead> <tbody> <tr> <td>Taxa do ofertante</td> <td>0.002 BTC</td> <td>0.20%</td> </tr> <tr> <td>Taxa do tomador</td> <td>0.006 BTC</td> <td>0.60%</td> </tr> </tbody> </table> <p class="note">Taxas de negociação de BSQ, por 1 BTC negociado:</p> <table> <thead> <tr> <th></th> <th>Taxas em BSQ</th> <th>Taxa como % da transação no valor de 1 BTC</th> </tr> </thead> <tbody> <tr> <td>Taxa do ofertante</td> <td>13 BSQ</td> <td>0.13%</td> </tr> <tr> <td>Taxa do tomador</td> <td>38 BSQ</td> <td>0.38%</td> </tr> </tbody> </table> </div> <p>As porcentagens de taxa em BSQ acima assumem uma taxa de 0.00001 BSQ/BTC.</p> <p>As taxas nominais de BTC e BSQ são ajustadas periodicamente pela votação da DAO para acomodar as flutuações do mercado.</p> <p>As taxas mínimas de negociação são fixadas em 0,00005 BTC e 0,03 BSQ para evitar limites com poeira.</p>
```
Comment (m52go) [inline _data/pt-BR.yml]: This FAQ isn't updated with current fees.

See:
https://bisq.network/faq/#cost-to-trade

File: _data/pt-BR.yml [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+      <li><a href='https://docs.bisq.network/payment-methods#account-signing' target="_blank">Assinatura de conta</a> faz com que meios de pagamento de alto risco sejam assinados ao verificar a integridade de um negociador antes que a idade da conta entre em vigor. Até que essas contas sejam assinadas, elas permanecem com o limite de 0.01 BTC.</li>
+      <li>Meios de pagamento fiat são removidos prontamente assim que descobrimos que são feitos estornos.</li>
+    </ol>
+  hReputationSystem: "Um sistema de reputação pura entre os negociadores não tornaria os mediadores e os árbitros desnecessários?"
+  pReputationSystem: >
+    <p>Reputação pura, sem a opção da arbitragem, é um sistema de proteção fraco, já que você nunca poderia evitar ataques do tipo <a href="https://en.wikipedia.org/wiki/Sybil_attack" target="_blank">sybil</a> ou <a href="https://en.wikipedia.org/wiki/Confidence_trick" target="_blank"> long con.</p> <p>Tais mecanismos também são problemáticos para proteger a privacidade e manter o serviço descentralizado.</p>
```
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
    <p>Reputação pura, sem a opção da arbitragem, é um sistema de proteção fraco, já que você nunca poderia evitar ataques do tipo <a href="https://en.wikipedia.org/wiki/Sybil_attack" target="_blank">sybil</a> ou <a href="https://en.wikipedia.org/wiki/Confidence_trick" target="_blank"> long con<a/>.</p> <p>Tais mecanismos também são problemáticos para proteger a privacidade e manter o serviço descentralizado.</p>
```

**Review by m52go — CHANGES_REQUESTED** [review]
Getting there, please see comments and suggested commits.

File: _data/pt-BR.yml (hunk 4 above)
Comment (FKrauss) [reply _data/pt-BR.yml]: fixed with changes in transifex and applied here, see commit 
72292a5

**Comment by FKrauss** [comment]
Alright I added a new commit that contains a few updated strings from Transifex. These were not reviewed there and were lagging. Now they should be correct.

**Comment by m52go** [comment]
@FKrauss thanks. I just fixed 1 minor issue in FAQ and also fixed a merge conflict.

One last thing, and then I think it's ready to merge. Is there any way you can think of to make the following strings smaller? They don't fit the layout.

In `pt-BR.yml`:
* `liAllDownloadsText: Ir para a página de downloads`
* `liArchText: "Repositório dos Usurários de Arch "`

Screenshot from 2019-12-02 12-08-26

**Comment by m52go** [comment]
ACK

**Comment by FKrauss** [comment]
All right, fixed these last two. and fixed them in transifex, so next time it won't cause the same problem again

**Pull request merged**